src/Symfony/Component/Messenger/Tests/HandleTraitTest.php

```diff
@@ -101,3 +118,19 @@ public function query($query): string
         return $this->handle($query);
     }
 }
+
+class TestCommandBus
```

Comment:
This is an edge case class and should be named more specifically according to its context.